### DISCUSSION TITLE
现在情况怎么样了啊 为啥网上看不到什么动静了

### DISCUSSION THREAD

**Discussion by YRApril** [WeNeedHome/SummaryOfLoanSuspension#1051] [General]
目前大家的情况怎么样了啊

**Comment by ghost** [comment]
🈚了

**Comment by ChinaChenMingQuan** [comment]
事实证明，国内不能团结真正的朋友，以攻击真正的敌人。

**Comment by Publius-thebard** [comment]
中央会议已经强调保交房了。根据中央意见，中房协正在协调金融机构成立2000-3000亿的基金来保障了

**Reply by MineCreeper86** [reply]
贴个链我去围观围观

**Reply by zhw173** [reply]
![image](https://user-images.githubusercontent.com/109256681/182019007-7b19e8af-d615-4b40-ab37-6d3b6a75ed19.png)

**Reply by zhw173** [reply]
3000亿房地产基金那个不知道真假

**Reply by jingzz1** [reply]
> ![image](https://user-images.githubusercontent.com/109256681/182019007-7b19e8af-d615-4b40-ab37-6d3b6a75ed19.png)

坚持房子是用来住的不是用来炒的……支持刚性和改善性住房需求……保交楼稳民生……这话我都听了十年了，然后深圳房价10年涨了10倍！

**Reply by TomMao-CN** [reply]
> 中央会议已经强调保交房了。根据中央意见，中房协正在协调金融机构成立2000-3000亿的基金来保障了

听其言，观其行

**Reply by Emic-Super** [reply]
想多了，怎么保证有这么多钱？哪来的钱？张嘴就来？谁来监督？都无

**Reply by tiancaipipi110** [reply]
‘中房协’+’正在’+’协调’+’金融机构’+‘基金‘ = 还是要百姓买单。

**Comment by zhishixiang** [comment]
老爷们都原谅你了你还想怎样

**Comment by Accelerator6666** [comment]
暗里酝酿的风暴，比张牙舞爪的风暴更可怕
